2025-T2/T2-HCD/projects/IRL/.env.example

```
@@ -0,0 +1,24 @@
+# Server Configuration
+NODE_ENV=development
+PORT=3000
+
+# Redis Configuration
+REDIS_HOST=localhost
+REDIS_PORT=6379
+REDIS_PASSWORD=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The REDIS_PASSWORD key should go before the REDIS_PORT key

(UnorderedKey)